security-framework/src/item.rs

```diff
@@ -175,7 +174,8 @@ impl ItemSearchOptions {
     /// Sets kSecAttrAccessGroup to kSecAttrAccessGroupToken
     #[inline(always)]
     pub fn access_group_token(&mut self) -> &mut Self {
-        self.access_group = unsafe { Some(CFString::wrap_under_get_rule(kSecAttrAccessGroupToken)) };
```

Comment:
Please don't change formatting of unrelated things

Reply:
Sorry!  I use `rustfmt` and it made some changes which I mistakenly assumed were expected.  I will revert any changes it made in older code.

Reply:
Reverted in my latest commits.